### DISCUSSION TITLE
How spaCy perform NER?

### DISCUSSION THREAD

**Discussion by teohsinyee** [explosion/spaCy#11278] [Help: Other Questions] [answered]
If all data were lowercase, will it affect NER?
Is casing important to NER?

I did simple experiment using [displaCy Named Entity Visualizer](https://explosion.ai/demos/displacy-ent?text=when%20jason%20started%20working%20on%20self-driving%20cars%20at%20google%20in%202007.&model=en_core_web_sm&ents=person%2Corg%2Cper%2Cmisc)

Cased sentence: when **Jason** started working on self-driving cars at **Google** in 2007.
Uncased sentence: when **jason** started working on self-driving cars at **google** in 2007.

Displacy able to identify Organization & Person from Cased sentence. But NO entities were identified from Uncased sentence.

I'm wondering why & how spaCy model identify 'Google' but can't identify 'google'?

**Comment by thomashacker** [comment, marked as answer]
Yes, the NER architecture uses features likes `NORM` and `SHAPE` which are both case-sensitive. The pre-trained models are not 100% accurate, and mistakes like these can happen. We have a [master thread](https://github.com/explosion/spaCy/issues/3052) for these cases.
